core/themes/basis/css/layout.css

```diff
@@ -11,3 +11,30 @@
 .layout .l-messages {
   margin: 0 0 2rem;
 }
+
+/* Push the footer down to the bottom of the page. */
+.layout {
+  box-sizing: border-box;
+  height: 100vh;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-flex-direction: column;
+  -ms-flex-direction: column;
+  flex-direction: column;
+}
+.admin-bar body .layout {
+  padding-top: 33px;
+  margin-top: -33px;
+}
```

Comment:
I copied these from the code in Bartik's style.css and have tried to understand the history behind them. It seems the values were raised from 30px to 33px in https://github.com/backdrop/backdrop/pull/876 having been introduced in https://github.com/backdrop/backdrop/pull/618, but I cannot see an explanation of why they are there or why the increase. Is there some explanation in these PRs?